app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -26,9 +26,28 @@
   <p class="margin-bottom-0"><%= t('in_person_proofing.body.barcode.deadline_restart') %></p>
 <% end %>
 
+<p><%= t('in_person_proofing.body.barcode.email_sent') %></p>
+
+<p>
+  <% if @presenter.service_provider_homepage_url.present? %>
+    <%= t(
+          'in_person_proofing.body.barcode.return_to_partner_html',
+          link: link_to(
+            t(
+              'in_person_proofing.body.barcode.return_to_partner_link',
+              sp_name: @presenter.sp_name,
+            ),
+            @presenter.service_provider_homepage_url,
+          ),
+        ) %>
+  <% else %>
+    <%= t('in_person_proofing.body.barcode.close_window') %>
+  <% end %>
```

Comment:
Would probably be good to include this conditional in [the ERB spec](https://github.com/18F/identity-idp/blob/main/spec/views/idv/in_person/ready_to_verify/show.html.erb_spec.rb).

Reply:
Added in [3583311](https://github.com/18F/identity-idp/pull/7526/commits/3583311e9439f38408b3d7fca6837eced11a0219)